ckanext/canada/i18n/fr/LC_MESSAGES/canada.po

```diff
@@ -788,6 +788,9 @@ msgstr "Veuillez entrer une année valide"
 msgid "Please enter a month number from 1-12"
 msgstr "Veuillez entrer une valeur comprise entre 1 et 12"
 
+msgid "The number of pages released must not be negative"
+msgtr "Le nombre de pages publiées ne doit pas être négatif"
```

Comment:
misspelling here (even github picked it up)

Reply:
Another thought: if you make this error message "This value must not be negative" then we can apply the same message and translation to other fields with the same validation check.

The UI and API will show the field name next to the error, e.g. "Number of Pages: The number of pages released must not be negative" is a little redundant.